### DISCUSSION TITLE
1.18 datapack map with negative camades request update on GriefPrevention

### DISCUSSION THREAD

**Discussion by comonier** [GriefPrevention/GriefPrevention#1552] [Ideas]
i have tried change
MaximumDepth: 0 to -63
but it still not working
on 0 -1 -2 -3 GriefPrevention not protect really

**Comment by bigpresh** [comment]
Yeah, it's quite possible that negative y coords are going to require some updates in GP (and a whole host of other plugins) to work properly.

At first glance, though, I don't see any obvious "oh yeah, that bit will choke on negative numbers" in the GP max depth code, though.

Can you share the config file you're having problems with, and more details on what actually happens?  ("still not working" or "not protect really" doesn't really give much info)
